specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/WebApps.json

```diff
@@ -1188,6 +1188,48 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/authsettings/validateUpgradePath": {
```

Comment:
Is appSettings in this API -> an instance of config or a nested resource? The reason I asked is that in below APIs, I see appSettings as a nested resource.